plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -57,6 +60,11 @@ open class KtlintPlugin : Plugin<Project> {
         val ktLintConfig = createConfiguration(target, extension)
         val multiplatformExtension = target.extensions.getByType(KotlinMultiplatformExtension::class.java)
 
+        multiplatformExtension.sourceSets.ifNotEmpty {
+            target.createMetaKtlintCheckTask()
+            target.createMetaKtlintFormatTask()
+        }
```

Comment:
What if the source set is empty when this plugin is evaluated but has sources added to it by a later plugin?